cfgov/jinja2/v1/youth_employment_success/route-details.html

```diff
@@ -0,0 +1,72 @@
+<div class="content-l content-l_col-2-3 yes-route-details">
+    <p class="h4 u-mt0">
```

Comment:
https://github.com/cfpb/cfgov-refresh/pull/5211#discussion_r321787909

Reply:
I commented on your comment,  adding the block means I still need to apply a `u-mbx` class to control the amount of padding on the bottom of the element